tests/test_pipermail.py

```diff
@@ -49,6 +50,14 @@ def read_file(filename, mode='r'):
     return content
 
 
+class PipermailListMocked(PipermailList):
```

Comment:
Do you think this is needed. I would raise an `OSError` exception. It's more clear than use `chmod` to raise that error.